classes/04class/exercises/c04-iac02/mrcsmonteiro/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "vpc_cidr" {
+  type = string
+}
+
+variable "vpc_name" {
+  type = string
+}
+
+variable "subnet_name_public_a" {
+  type = string
+}
+
+variable "subnet_cidr_public_a" {
+  type = string
+}
+
+variable "subnet_name_public_b" {
+  type = string
+}
+
+variable "subnet_cidr_public_b" {
+  type = string
+}
+
+variable "subnet_name_private_a" {
+  type = string
+}
+
+variable "subnet_cidr_private_a" {
+  type = string
+}
+
+variable "subnet_name_private_b" {
+  type = string
+}
+
+variable "subnet_cidr_private_b" {
+  type = string
+}
```

Comment:
tip: it is possible to group this inputs using other data types, as map, list and objects. It simplifies the input as some Regions will have more than 3 AZs and deployments more than 3 subnets too